src/Domain/Aggregate/PathAggregate/NullPathAggregate.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Domain\Aggregate\PathAggregate;
+
+final class NullPathAggregate implements PathAggregateInterface
```

Comment:
Ooh, I'm glad you asked this... I created it for domain classes to use to avoid dependence on higher layers, but I didn't end up using it in the Domain. So I can just replace instances of it with `PathAggregate`. Done.